core/services/workflow_service/models/project.py

```diff
@@ -0,0 +1,23 @@
+from uuid import UUID
+from sqlalchemy import Column, String, DateTime
+from sqlalchemy.orm import relationship
+
+import uuid
+
+from datetime import datetime
+from utils.database.connection import Base
+
+
+class Project(Base):
+    __tablename__ = "projects"
+
+    uuid = Column(UUID(as_uuid=True), primary_key=True,
+                  default=uuid.uuid4)
+    name = Column(String(100), nullable=False)
+    created_at = Column(DateTime, default=datetime.utcnow, nullable=False)
+
+    users = relationship("User", secondary="user_project",
+                         back_populates="projects")
```

Comment:
I am not 100% sure. But I believe `back_populate` here only works if you also add a `projects = relationship(..)` to the `user` model. 

What do you think? This is what I found from the docs:

`relationship()` Provide a relationship between *two* mapped classes.
`back_populates` Indicates the name of a [relationship()](https://docs.sqlalchemy.org/en/20/orm/relationship_api.html#sqlalchemy.orm.relationship) on the related class that will be *synchronized* with this one. It is usually expected that the [relationship()](https://docs.sqlalchemy.org/en/20/orm/relationship_api.html#sqlalchemy.orm.relationship) on the related class also refer to this one. (...)

Reply:
Yes, you're right, I forgot, that I didn't put a projects relationship in User. I'll either remove the back_populates or add the relationship.